### DISCUSSION TITLE
Commute-Telecommute Frequency Model

### DISCUSSION THREAD

**Discussion by Ennazus** [synthicity/activitysim#695] [Ideas]
Hi! First time poster-long time friend here,
In recent years, we've all observed the complexities around telecommuting, ranging from people who always telecommute, to people who telecommute and commute everyday.

For people who commute and telecommute sometimes, it may be more important to capture the frequency of commuting  and telecommuting jointly, as opposed to the frequency of telecommuting. The reason is that people do not at all one-to-one substitute telecommuting for commuting. I've done a bunch of summarization and modeling around the topic that I'd be happy to share. Vehicle Miles Traveled on a day is more associated and significant with commute frequency than telecommute frequency.
Bye!

**Comment by DavidOry** [comment]
The Ohio 3C models do explicitly model commute frequency and telecommute frequency jointly (choice set below), which then informs the activity pattern models, which then informs tour generation, which eventually informs VMT.  This gets at both part-time work and telecommuting. Does your research suggest whether these alternatives are sufficient? Or do we need to capture less frequent telecommuting? I'd be interested to see a frequency distribution across these choices pre- and post-COVID.  
thanks!

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/wsp/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/wsp/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



alt | numDaysCommuting | numDaysTelecommuting
-- | -- | --
1 | 1 | 0
2 | 1 | 1
3 | 1 | 2
4 | 1 | 3
5 | 1 | 4
6 | 2 | 0
7 | 2 | 1
8 | 2 | 2
9 | 2 | 3
10 | 3 | 0
11 | 3 | 1
12 | 3 | 2
13 | 4 | 0
14 | 4 | 1
15 | 5 | 0
16 | 0 | 5



</body>

</html>
